plugins/att_cybersecurity_alienvault_otx/unit_test/util.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+from komand_att_cybersecurity_alienvault_otx.connection.connection import Connection
+from komand_att_cybersecurity_alienvault_otx.connection.schema import Input
+import json
+
+
+class Util:
+    @staticmethod
+    def default_connector(action):
+        default_connection = Connection()
+        default_connection.logger = logging.getLogger("connection logger")
+        params = {
+            Input.API_KEY: {"secretKey": "3395856ce81f2b7382dee72602f798b642f14140"},
```

Comment:
This is not a real API key, correct? Just making sure :)

Reply:
That's right, it's not a real API key :)